packages/@contentlayer/utils/src/node/fs.ts

```diff
@@ -95,6 +95,20 @@ export const mkdirp = (dirPath: string): T.Effect<OT.HasTracer, MkdirError, void
     ),
   )
 
+export const rm = (path: string): T.Effect<OT.HasTracer, RmError | FileNotFoundError, void> =>
+  OT.withSpan('rm', { attributes: { path } })(
+    T.tryCatchPromise(
+      () => fs.rm(path, { recursive: true }),
+      (error: any) => {
+        if (error.code === 'ENOENT') {
+          return new FileNotFoundError({ filePath: path })
```

Comment:
Since the node `rm` function accepts file or dir, should we create a new `FileOrDirNotFoundError` instead ?

Reply:
> Since the node `rm` function accepts file or dir, should we create a new `FileOrDirNotFoundError` instead ?

Yeah, that probably makes sense.



> Also, I'm wondering if an additional `rmForce` function that calls `fs.rm` with `{force:true}` would make sense?

I'd probably rather make this an optional parameter for `rm`. WDYT?

Reply:
> I'd probably rather make this an optional parameter for rm. WDYT?

Yeah, that's fine with me. I pushed the changes :)